addons/point_of_sale/tests/test_frontend.py

```diff
@@ -798,6 +798,11 @@ def test_pos_closing_cash_details(self):
         current_session = self.main_pos_config.current_session_id
         current_session.post_closing_cash_details(0)
         current_session.close_session_from_ui()
+        self.pos_user.write({
+            'groups_id': [
+                (4, self.env.ref('account.group_account_user').id),
+            ]
+        })
```

Comment:
we want the pos_user in the test to not be different than the actual pos_user.
The solution here would be to put the step in a tour that runs as pos_admin, but seeing as we don't have any such tour in standard pos, i suggest to just leave the `test_report_sesison.py` for now. You can remove the step from the tour

Reply:
The `test_report_session.py` checks the PDF content and rendering. Removing the tour step we could miss errors related to the controller calls, or OWL issues.
I have removed the step from the tour, but I wanted to highlight these potential issues if I am not wrong, for your consideration.

Thanks for the feedback.